.fixtures.yml

```diff
@@ -1,4 +1,5 @@
 ---
 fixtures:
-  repositories:
-    stdlib: https://github.com/puppetlabs/puppetlabs-stdlib.git
+  forge_modules:
```

Comment:
Typically we consume these from git to see about future failures. It's not great because you could be testing for too new versions, but the `forge_modules` don't take `metadata.json` into account either.

Reply:
OK, for me... I've changed that to the repos